opendebates/models.py

```diff
@@ -125,13 +157,13 @@ def get_absolute_url(self):
 
     def my_tweet_text(self):
         params = {
-            "hashtag": settings.SITE_THEME['HASHTAG'],
+            "hashtag": self.site_mode.hashtag,
         }
         return _(u"Vote for my progressive idea for @ThinkBigUS #%s(hashtag)s. "
                  "30 leaders in Congress will see top ideas!" % params)
 
     def tweet_text(self):
-        text = settings.SITE_THEME['TWITTER_QUESTION_TEXT']
+        text = self.site_mode.twitter_question_text
         if self.voter.twitter_handle:
             text += u" h/t @%s" % self.voter.twitter_handle
```

Comment:
While we're in here -- I know Lilia also wants to be able to adjust this `"h/t"` suffix behavior too, and put the submitter's twitter handle within the body of the tweet instead of at the end.  ("Vote on @SO-AND-SO's question!")  But that gets complicated by the fact that twitter handle is optional, so there has to be some branching logic somewhere.  What would you think about having two fields on site mode here instead -- `twitter_question_text_with_handle` and `twitter_question_text_no_handle`?   Then this method would be something like

```
  if self.voter.twitter_handle:
    return self.site_mode.twitter_question_text_with_handle.format(handle=self.voter.twitter_handle)
  return self.site_mode.twitter_question_text_no_handle
```